framework/decode/parsed_block.h

```diff
@@ -73,10 +73,9 @@ class ParsedBlock
     enum BlockState
     {
         kInvalid = 0,        // Set on read error (typically block size doesn't match expected parsed size)
-        kUnknown,            // Set when block is of an unknown type (no parsing done beyond header)
+        kRawBlock,           // Set when block is of an unknown type (no parsing done beyond header)

```

Comment:
The Raw block doesn't communicate "unknown" as well as the old value in my opinion.  In fact this seems confusing because it indicates a "decompressed" block in my verbage when I read this.